src/foundry/client/applications/ux/draggable.d.mts

```diff
@@ -1,9 +1,148 @@
+import type Application from "#client/appv1/api/application-v1.mjs";
+import type ApplicationV2 from "#client/applications/api/application.d.mts";
+
 /**
  * A UI utility to make an element draggable.
- * @remarks TODO: Stub, copy from v12 implementation & update
  */
-declare class Draggable {}
+declare class Draggable<R extends boolean | Draggable.Resizable = false> {
+  /**
+   * @param app       - The Application that is being made draggable.
+   * @param element   - The application's outer-most element.
+   * @param handle    - The element that acts as a drag handle. Supply false to disable dragging.
+   * @param resizable - Is the application resizable? Supply an object to configure resizing behavior
+   *                    or true to have it automatically configured.
+   */
+  constructor(app: Draggable["app"], element: JQuery | HTMLElement, handle: Draggable["handle"], resizable?: R);
+
+  /**
+   * The Application being made draggable.
+   */
+  app: Application.Any | ApplicationV2.Any;
+
+  /**
+   * The Application's outer-most element.
+   */
+  element: HTMLElement;
+
+  /**
+   * The drag handle, or false to disable dragging.
+   */
+  handle: HTMLElement | boolean;
+
+  /**
+   * Registered event handlers.
+   * @defaultValue `{}`
+   */
+  // prettier-ignore
```

Comment:
I'd probably say this regardless but this is a good sign that you should extract this into a helper type. It'd also help distributivity.